[ci:docs] Remove future tense from man pages (#18559)
containers/podman: docs/source/markdown/podman-generate-systemd.1.md

- Note: The generated `podman run` command contains an `--sdnotify` option with the value taken from the container.
If the container does not have any explicitly set value or the value is set to __ignore__, the value __conmon__ is used.
The reason for overriding the default value __container__ is that almost no container workloads send notify messages.

> Systemd waits for a ready message that never comes, if the value __container__ is used for a container

Reviewer: I think "will wait" is better than "waits" in this context, but it is a coinflip.

```suggestion
Systemd will wait for a ready message that never comes if the value __container__ is used for a container
```

Set environment variables to the systemd unit files.

> If an environment variable is specified without a value, Podman checks the host environment for a value and set the variable only if it is set on the host. As a special case, if an environment variable ending in __*__ is specified without a value, Podman searches the host environment for variables starting with the prefix and adds those variables to the systemd unit files.

Reviewer: ```suggestion
If an environment variable is specified without a value, Podman checks the host environment for a value and sets the variable only if it is set on the host. As a special case, if an environment variable ending in __*__ is specified without a value, Podman searches the host environment for variables starting with the prefix and adds those variables to the systemd unit files.
```

#### **--files**, **-f**

Generate files instead of printing to stdout.  The generated files are named {container,pod}-{ID,name}.service and are placed in the current working directory.

Note: On a system with SELinux enabled, the generated files inherits contexts from the current working directory. Depending on the SELinux setup, changes to the generated files using `restorecon`, `chcon`, or `semanage` may be required to allow systemd to access these files. Alternatively, use the `-Z` option when running `mv` or `cp`.

#### **--format**=*format*

> Print the created units in specified format (json). If `--files` is specified the paths to the created files is printed instead of the unit content.

Reviewer: ```suggestion
Print the created units in the specified format (json). If `--files` is specified, the paths to the created files are printed instead of the unit content.
```

#### **--name**, **-n**

Use the name of the container for the start, stop, and description in the unit file

#### **--new**

> Using this flag yields unit files that do not expect containers and pods to exist.  Instead, new containers and pods are created based on their configuration files.  The unit files are created best effort and may need to be further edited; please review the generated files carefully before using them in production.

Reviewer: ```suggestion
This flag yields unit files that do not expect containers and pods to exist.  Instead, new containers and pods are created based on their configuration files.  The unit files are created best effort and may need further editing; please review the generated files carefully before using them in production.
```

Set the systemd restart policy.  The restart-policy must be one of: "no", "on-success", "on-failure", "on-abnormal",
"on-watchdog", "on-abort", or "always".  The default policy is *on-failure* unless the container was created with a custom restart policy.

> Note that generating a unit without `--new` on a container with a custom restart policy can lead to issues on shutdown; systemd attempts to stop the unit while Podman tries to restart it.  It is recommended to to create the container without `--restart` and use the `--restart-policy` option instead when generating the unit file.

Reviewer: ```suggestion
Note that generating a unit without `--new` on a container with a custom restart policy can lead to issues on shutdown; systemd attempts to stop the unit while Podman tries to restart it.  Creating the container without `--restart` and using the `--restart-policy` option when generating the unit file is recommended.
```

Add template specifiers to run multiple services from the systemd unit file.

> Note that if `--new` was not set to true, it is set to true by default. However, if `--new` is set to `false` explicitly the command fails.

Reviewer: ```suggestion
Note that if `--new` was not set to true, it is set to true by default. However, the command fails if `--new` is set to `false` explicitly.
```

#### **--wants**=*dependency_name*

Add the systemd unit wants (`Wants=`) option, that this service is (weak) dependent on. This option may be specified more than once. This option does not influence the order in which services are started or stopped.

> User-defined dependencies are appended to the generated unit file, but any existing options such as needed or defined by default (e.g. `online.target`) is **not** removed or overridden.

Reviewer: ```suggestion
User-defined dependencies are appended to the generated unit file, but any existing options needed or defined by default (e.g., `online.target`) are **not** removed or overridden.
```
